### PR TITLE
chore: add support for 3.3.0-RC1

### PR DESCRIPTION
**Pull request by ckipp01**

### CODE REVIEW

**Review by tgodzik — APPROVED** [review]
I haven't noticed it's out, thanks and LGTM!

**Comment by ckipp01** [comment]
> I haven't noticed it's out, thanks and LGTM!

I'm getting some odd failures with ivy completions. I think there is something funky with it and RC versions of Scala. So I'm trying to tackle that first.

**Comment by tgodzik** [comment]
> > I haven't noticed it's out, thanks and LGTM!
> 
> I'm getting some odd failures with ivy completions. I think there is something funky with it and RC versions of Scala. So I'm trying to tackle that first.

I think you might need to add it to quick publish list.

**Comment by ckipp01** [comment]
> > > I haven't noticed it's out, thanks and LGTM!
> > 
> > 
> > I'm getting some odd failures with ivy completions. I think there is something funky with it and RC versions of Scala. So I'm trying to tackle that first.
> 
> I think you might need to add it to quick publish list. Meaning `quickPublishScalaVersions`

ahhhh dang, I totally missed that!!

File: project/V.scala [hunk 1]
```diff
@@ -6,6 +6,7 @@ object V {
   val scala212 = "2.12.17"
   val scala213 = "2.13.10"
   val scala3 = "3.2.2"
+  val scala3RC = "3.3.0-RC1"
```
Comment (tgodzik) [inline project/V.scala:9]: Maybe we could have:
```suggestion
  val scala3RC = Option("3.3.0-RC1")
```
which would avoid issues with removing/readding it?

**Comment by ckipp01** [comment]
what the heck.... it merged before the run was finished..... I didn't manually do it. Did the auto-merge screw up here?

File: project/V.scala (hunk 1 above)
Comment (ckipp01) [reply project/V.scala:9]: Ah good idea, I'll add this.

**Comment by tgodzik** [comment]
> what the heck.... it merged before the run was finished..... I didn't manually do it. Did the auto-merge screw up here?

Yeah, automerge only waits for the required jobs, which is scalafix and scalafmt, since otherwise other maintainers would have issues merging, since you need admin override.

**Comment by ckipp01** [comment]
> Yeah, automerge only waits for the required jobs, which is scalafix and scalafmt, since otherwise other maintainers would have issues merging, since you need admin override.

Huh, good to know. I won't use it again 😆

**Pull request merged**